bindings/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "bindings"
```

Comment:
morally this is just the `wasi` crate. could we just name it that, with `publish=false`? And I think in that case we want to version it `0.12.0+preview2`?

Reply:
i thought so. i pushed all the changes based on your suggestion